test/ModuleInterface/modifiers.swift

```diff
@@ -40,4 +40,15 @@ public final class FinalClass {
     }
     @inlinable set {}
   }
+
+  // CHECK: @objc dynamic public var d: [[INT]] {
+  // CHECK-NEXT: {{^}} @objc get
+  // CHECK-NEXT:   set[[NEWVALUE]] {}
+  // CHECK-NEXT: }
+  public @objc dynamic var d: Int {
```

Comment:
Uh. You shouldn't be able to mix attributes in the middle of modifiers like this…

Also, please do add tests for `mutating get` and `nonmutating set`, now that it's come up.